packages/vue/src/components/molecules/SfSection/SfSection.stories.js

```diff
@@ -14,10 +14,6 @@ storiesOf("Molecules|Section", module)
       customClass: {
```

Comment:
you can also delete both customClasses in this file :) we don't need them anymore

Reply:
done

Reply:
> Looks great! Just a few fixes

Fixed the 2 last comments